app/models/ticket.js

```diff
@@ -44,5 +44,13 @@ export default ModelBase.extend({
 
   itemTotal: computed('price', 'quantity', function() {
     return this.price * this.quantity;
+  }),
+
+  taxIncurred: computed('event.tax', function() {
```

Comment:
This name is confusing. It looks like we are calculating tax incurring on each ticket but this is amount after adding tax. Something like `taxedAmount` or `ticketPriceWithTax` ot `finalPrice` would be good IMO.

Reply:
@pradeepgangwar `ticketPriceWithTax` it is then :smile: